lib/SPIRV/SPIRVReader.cpp

```diff
@@ -3658,9 +3661,10 @@ void SPIRVToLLVM::transIntelFPGADecorations(SPIRVValue *BV, Value *V) {
     Type *Int8PtrTyPrivate = Type::getInt8PtrTy(*Context, SPIRAS_Private);
     IntegerType *Int32Ty = Type::getInt32Ty(*Context);
 
-    llvm::Constant *Fields[4] = {
+    llvm::Constant *Fields[5] = {
```

Comment:
You can remove array size to avoid this line change in the future (if the signature is changed again)

Reply:
That sounds good, but it will require some style alignment, for example in `transUserSemantic` there is `llvm::Constant *Fields[5] = {`. If we want to remove the sizes - it's better to do in a single NFC patch.

Reply:
Well, actualy, I've started leaning to appose this suggestion. If/when signature changes again, we would need to change this array anyway, and may be with the size set it would be harder to pass by this code during initial eye-debugging.